components/AppBar/src/ColorThemer/MDCAppBarColorThemer.m

```diff
@@ -23,4 +25,15 @@ + (void)applyColorScheme:(id<MDCColorScheme>)colorScheme
   appBar.headerViewController.headerView.backgroundColor = colorScheme.primaryColor;
 }
 
++ (void)applyExperimentalColorScheme:(MDCExperimentalColorScheme *)colorScheme toAppBar:(MDCAppBar *)appBar {
+  appBar.headerViewController.headerView.backgroundColor = colorScheme.backgroundColor;
+  appBar.headerViewController.headerView.tintColor = colorScheme.textColor;
```

Comment:
tintColor is dangerous.  If we completely own this class does it make sense to define an explicitly UI_APPEARANCE_SELECTOR property on headerView?

Reply:
We could, but since MDCAppBar can't support UIAppearance, I don't think it will help things here.  If we did that and used class property defaults for other properties, I think the mix of color assignments will be a bit harder to reason about.  Some will be applied in `-init` and others will be applied by UIAppearance only once the view is in a window.